subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/configurationcache/ConfigurationCacheProblemsFixture.groovy

```diff
@@ -194,6 +191,19 @@ final class ConfigurationCacheProblemsFixture {
         return spec
     }
 
+    static void assertHtmlReportHasProblems(
```

Comment:
```suggestion
    static void assertHtmlReportProblems(
```

I think this might suit the case of the report having no problems slightly better, in the future 🙃